outputs.tf

```diff
@@ -47,6 +47,14 @@ output "table_ids" {
   description = "Unique id for the table being provisioned"
 }
 
+output "computed_table_ids" {
```

Comment:
nit: computed is somewhat of an internal term. WDYT about table_fqns `Fully qualified names for the table with format projects/{{project}}/datasets/{{dataset}}/tables/{{name}}`

Reply:
Thanks! Souds good. @amrkk123 Please rename this to `table_fqns` and the description.